Views/PlaceResultCell.h

```diff
@@ -8,10 +8,13 @@
 
 #import <UIKit/UIKit.h>
 #import <GooglePlaces/GooglePlaces.h>
+#import "Place.h"
```

Comment:
Are you using this import here? I think it's safe to remove?

Reply:
lmk if it's not (I think we should move it back into PlaceResultCell.m) and we can figure out why :)